api-samples/README.md

```diff
@@ -0,0 +1,6 @@
+Before running the code samples, ensure your development environment is
```

Comment:
I would add "developer account" here - it's not just their development environment that needs to be configured.